README.md

```diff
@@ -21,7 +21,7 @@ make
 Alternatively a Dockerfile is supplied:
 
 ```sh
-docker run -p 9150:9150 quay.io/prometheus/memcached-exporter:v0.7.0
+docker run -p 9150:9150 quay.io/prometheus/memcached-exporter:latest
```

Comment:
Should we add a note on top explaining that this tag returns the current main branch version, and that people should check quay/docker hub for the list of currently available tags?

Reply:
This returns the latest current release, not the main branch. We publish that under `:master`.

Reply:
Oh, even better then!